src/main/java/org/usfirst/frc/team5687/robot/OI.java

```diff
@@ -28,8 +32,12 @@ public OI() {
         joystick = new Joystick(1);
 
         JoystickButton reverseButton = new JoystickButton(gamepad, REVERSE);
+        JoystickButton bowlButton = new JoystickButton(joystick, BOWL);
 
+        // Drive Train Commands
         reverseButton.whenPressed(new ReverseDrive());
+        // Shooter Commands
+        bowlButton.whenPressed(new Bowl());
     }
```

Comment:
Should this be whilePressed() since the command doesn't have anything for isFinished?